test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -1,10 +1,22 @@
+import BaseFormObject from './components/BaseFormObject';
+
 export default class BasePageObject {
   constructor(t) {
     this.t = t;
     this.url = '';
+    // Not all pages may have a form
```

Comment:
I was wrong and actually all page has Form (currently all page extend from BaseView) thus doesn't need the comment.

Reply:
Okay will remove that.